src/extractors/extractors/src/utils.network.ts

```diff
@@ -15,7 +15,7 @@ export const performGetRequest = <T>(uri: string, json = true, headers?: Headers
     request({uri, headers, json}, (error, response, body) => {
       if (error || response.statusCode !== 200) {
         // For the purpose of HTTP requests made in this package, 403 and 404 both can mean an unauthorized request.
-        if (response.statusCode === 403 || response.statusCode === 404) {
+        if (response && (response.statusCode === 403 || response.statusCode === 404)) {
```

Comment:
these should be descriptive constants probably, but otherwise LGTM

Reply:
@dbuscaglia would you mind expanding on that? would love to address this before merging.